test/datatable.jl

```diff
@@ -286,25 +286,48 @@ module TestDataTable
         @test nothing == describe(f, NullableCategoricalArray(Nullable{String}["1", "2", Nullable()]))
     end
 
-    dt = DataTable(Fish = CategoricalArray(["Bob", "Bob", "Batman", "Batman"]),
-                   Key = ["Mass", "Color", "Mass", "Color"],
-                   Value = ["12 g", "Red", "18 g", "Grey"])
-    # Check that reordering levels does not confuse unstack
-    levels!(dt[1], ["XXX", "Bob", "Batman"])
-    #Unstack specifying a row column
-    dt2 = unstack(dt, :Fish, :Key, :Value)
-    #Unstack without specifying a row column
-    dt3 = unstack(dt, :Key, :Value)
-    #The expected output
-    dt4 = DataTable(Fish = ["Bob", "Batman"],
-                    Mass = ["12 g", "18 g"],
-                    Color = ["Red", "Grey"] )
-    @test isequal(dt2, dt4)
-    @test isequal(dt3, dt4)
-    # can't assign Nullable() to a typed column
-    #Make sure unstack works with NULLs at the start of the value column
-    # dt[1,:Value] = Nullable()
-    dt2 = unstack(dt,:Fish, :Key, :Value)
+    @testset "unstacking and nullables" begin
```

Comment:
Same here. It's great to move tests to testsets, but this shouldn't be made in a large PR which also changes the behavior of the API and its test. Now I (and future readers) can't see what changes in the tests and what is just a reorganization.